### PR TITLE
added issue reference to todos

### PR DESCRIPTION
**Pull request by vendz**
in reference to #2119

### CODE REVIEW

**Comment by vendz** [comment]
@googlebot I signed it!

**Comment by vendz** [comment]
@kwalrath please review my PR

**Review by kwalrath — COMMENTED** [review]
Thank you, @vendz!

I believe that the test failures are due to line length > 80. Could you break up the lines? While you're at it, you might make these full URLs. Or not. :)

**Comment by vendz** [comment]
@kwalrath sorry for the late reply as I am from India. Yeah I can do that

**Comment by vendz** [comment]
@kwalrath I have made changes to the TODO line so if you could review my PR now

**Comment by vendz** [comment]
@parlough i have made the changes you requested in this PR itself so if you could review it

File: null_safety_examples/misc/lib/tutorial/stream_interface.dart [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // ignore_for_file: annotate_overrides, type_annotate_public_apis, strict_raw_type, use_function_type_syntax_for_parameters
 import 'dart:async';
 
-// TODO(chalin): I believe that handleError's test argument should be declared
+// TODO(chalin): I believe that handleError's test argument should be declared (#3298)
 // as follows, but it isn't: {bool Function(dynamic error) test}.
```
Comment (parlough) [inline null_safety_examples/misc/lib/tutorial/stream_interface.dart:5]: Can you instead put a link to the issue so its easier to access when navigating the source?

```suggestion
// TODO(chalin): I believe that handleError's test argument should be declared
// as follows, but it isn't: {bool Function(dynamic error) test}.
// (https://github.com/dart-lang/site-www/issues/3298)
```

File: null_safety_examples/misc/test/language_tour/browser_test.dart [hunk 2]
```diff
@@ -13,7 +13,8 @@ void main() {
     document.body?.appendHtml(div);
     void reverseText(MouseEvent e) {}
 
-    // TODO(miquelbeltran) Maybe ! would be better here
+    // TODO(miquelbeltran) Maybe ! would be better here (#3295)
```
Comment (parlough) [inline null_safety_examples/misc/test/language_tour/browser_test.dart:16]: Including the issue link here would be great as well. 

```suggestion
    // TODO(miquelbeltran) Maybe ! would be better here
    // (https://github.com/dart-lang/site-www/issues/3295)
```

**Review by parlough — APPROVED** [review]
Some minor suggestions, feel free to ignore them if desired.

Thanks again!

**Comment by vendz** [comment]
@parlough made the changes you requested.

**Comment by parlough** [comment]
@vendz Thanks for all the work, looks good to me! I'm running the checks and if they're all green, I'll merge these changes :)

File: null_safety_examples/misc/lib/language_tour/functions.dart [hunk 3]
```diff
@@ -26,7 +26,7 @@ void miscDeclAnalyzedButNotTested() {
   }
 
   {
-    // TODO(miquelbeltran) language-tour.md should explain that non-required params can be null
+
```
Comment (parlough) [inline null_safety_examples/misc/lib/language_tour/functions.dart:29]:
```suggestion
```

File: null_safety_examples/misc/test/language_tour/browser_test.dart [hunk 4]
```diff
@@ -14,6 +14,8 @@ void main() {
     void reverseText(MouseEvent e) {}
 
     // TODO(miquelbeltran) Maybe ! would be better here
+    // (https://github.com/dart-lang/site-www/issues/3295)
+
```
Comment (parlough) [inline null_safety_examples/misc/test/language_tour/browser_test.dart:18]:
```suggestion
```

**Review by parlough — CHANGES_REQUESTED** [review]

**Review by parlough — APPROVED** [review]

**Comment by parlough** [comment]
All tests were green after I removed the extra whitespace and the changes have been committed to the main branch(https://github.com/dart-lang/site-www/commit/c95f9bb1fee6029e7429540417318b0754ee172d). Thanks again and good work and communication around your first contribution :D

**Pull request merged**